pyproject.toml

```diff
@@ -25,8 +25,8 @@ typeCheckingMode = "strict"
 profile = "black"
 
 [tool.ruff]
-select = ["E", "F", "W", "I"]
-ignore = ["E741"]
+select = ["E", "F", "W", "I", "UP"]
+ignore = ["E741", "UP006", "UP007", "UP02", "UP032", "UP034", "UP035", "UP038"]
```

Comment:
Ignoring rules for now that will cause huge diff. Will add them back gradually to minimize changes.